tests/test_heat_fridge_alarms.sql

```diff
@@ -0,0 +1,3 @@
+SELECT *
+FROM {{ ref('heat_fridge_alarms') }}
+LIMIT 1
```

Comment:
question for my understanding: Is this test basically asserting that heat fridge alarms aren't created for our test data?